Emby.Drawing/ImageProcessor.cs

```diff
@@ -199,8 +208,13 @@ public async Task<(string Path, string? MimeType, DateTime DateModified)> Proces
             {
                 if (!File.Exists(cacheFilePath))
                 {
+                    // Limit number of parallel (more precisely: concurrent) image encodings to prevent a high memory usage
+                    await _parallelEncodingLimit.WaitAsync().ConfigureAwait(false);
+
                     string resultPath = _imageEncoder.EncodeImage(originalImagePath, dateModified, cacheFilePath, autoOrient, orientation, quality, options, outputFormat);
 
+                    _parallelEncodingLimit.Release();
```

Comment:
Shouldn't this be in a try-finally block?

Reply:
Yes, it should be in a try-finally block. Fixed.